ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/CVEsTableContainer.tsx

```diff
@@ -58,16 +59,22 @@ function CVEsTableContainer({
 
     const { data: imageCountData } = useQuery(unfilteredImageCountQuery);
 
-    const selectedCves = useSet<string>();
+    const selectedCves = useMap<string, ExceptionRequestModalProps['cves'][number]>();
     const [exceptionRequestModalOptions, setExceptionRequestModalOptions] =
         useState<ExceptionRequestModalOptions>(null);
 
     function openDeferralModal() {
-        setExceptionRequestModalOptions({ type: 'DEFERRAL', cves: selectedCves.asArray() });
+        setExceptionRequestModalOptions({
+            type: 'DEFERRAL',
+            cves: Array.from(selectedCves.values()),
```

Comment:
For what it is worth, ordinary ECMAScript methods are easier to read than the former helper function.